### PR TITLE
Allow updating NFT posts.

### PR DESCRIPTION
**Pull request by mattfoley8**

### CODE REVIEW

File: lib/constants.go [hunk 1]
```diff
@@ -986,7 +992,8 @@ var TestnetForkHeights = ForkHeights{
 	DeSoUnlimitedDerivedKeysBlockHeight: uint32(467217),
 
 	// FIXME: Set to real block height when we're ready.
-	AssociationsBlockHeight: math.MaxUint32,
+	AssociationsBlockHeight:          math.MaxUint32,
+	AllowUpdatingNFTPostsBlockHeight: math.MaxUint32,
```
Comment (lazynina) [inline lib/constants.go:996]: we'll combine these block heights - but that can happen separately.

**Review by lazynina — APPROVED** [review]

File: lib/constants.go (hunk 1 above)
Comment (mattfoley8) [reply lib/constants.go:996]: Resolved that in a subsequent PR: https://github.com/deso-protocol/core/pull/451.

File: lib/network.go [hunk 2]
```diff
@@ -5608,8 +5608,8 @@ func (tsl *TransactionSpendingLimit) Copy() *TransactionSpendingLimit {
 	}
 
 	if tsl.AssociationLimitMap != nil {
-		// Before the AssociationsBlockHeight, this map will be null.
-		// So we should ensure this is the case in the copy too.
+		// Before the AccessGrouopsAndAssociationsBlockHeight, this map will
```
Comment (mattfoley8) [inline lib/network.go:5611]:
```suggestion
		// Before the AccessGroupsAndAssociationsBlockHeight, this map will
```

**Comment by mattfoley8** [comment]
These changes have been merged into main separately.

**Pull request closed without merging**